skimage/transform/tests/test_radon_transform.py

```diff
@@ -294,13 +326,22 @@ def check_radon_iradon_circle(interpolation, shape, output_size):
     np.allclose(reconstruction_rectangle, reconstruction_circle)
 
 
-def test_radon_iradon_circle():
-    shape = (61, 79)
-    interpolations = ('nearest', 'linear')
-    output_sizes = (None, min(shape), max(shape), 97)
-    for interpolation, output_size in itertools.product(interpolations,
-                                                        output_sizes):
-        yield check_radon_iradon_circle, interpolation, shape, output_size
+# this might be overkill:
```

Comment:
Indeed. Mad skills though :).
I'd prefer to put more code here, but stay as clear as possible.

Reply:
My thinking was that sometime later one would want to test more than one shape and thus the complication. If you think it won't happen, I could go back to having one shape only as was before.

Reply:
In general, write the simplest code that gets the job done.  We can always refactor later.

Reply:
So what do you say about:
```python
shapes_radon_iradon_circle = ((61, 79), )
interpolations = ('nearest', 'linear')
output_sizes = (None,
                min(shapes_radon_iradon_circle[0]),
                max(shapes_radon_iradon_circle[0]),
                97)
```

Reply:
That looks fine to me.

Reply:
OK, I pushed this fix too.